manifest-el9-shared.yaml

```diff
@@ -0,0 +1,35 @@
+# Place here configuration that should happen on all el9* builds
+
+#zram default config is in a subpackage in c10s
+# Meanwhile, remove the default config from the package
+remove-from-packages:
+  # zram-generator-0.3.2 (shipped in centOS 9) provides a default
+  # zram-generator config, we want to disable it
+  - - zram-generator
+    - "/usr/lib/systemd/zram-generator.conf"
```

Comment:
Are there any consequences of running this on RHEL 9.4 and RHEL 9.6?

Reply:
The changes to the manifests here should provide no functional change IIUC..

Previously it was getting included on both via:

```
- if: osversion == "c9s"        
  include: zram-no-defaults.yaml
- if: osversion == "rhel-9.4"   
  include: zram-no-defaults.yaml
- if: osversion == "rhel-9.6"   
  include: zram-no-defaults.yaml

```